commit 5045279aa30641ba15dfd5cd34180ed1a058f7d1
Author: Mario Haustein <mario.haustein@hrz.tu-chemnitz.de>
Date:   Sat Aug 23 12:42:08 2025 +0200
Upstream: https://github.com/pololu/pololu-tic-software/pull/14

    Fix dectection of system libyaml
    
     * `libyaml` installs its .pc-file as `yaml-0.1` instead of `yaml`
     * the variable prefix in the `pkg_check_modules` was wrong `YAML`
       instead of `LIBYAML`

--- a/lib/CMakeLists.txt
+++ b/lib/CMakeLists.txt
@@ -11,11 +11,11 @@ if (NOT BUILD_SHARED_LIBS)
 endif ()
 
 if (USE_SYSTEM_LIBYAML)
-  pkg_check_modules(YAML REQUIRED yaml)
+  pkg_check_modules(LIBYAML REQUIRED yaml-0.1)
   string (REPLACE ";" " " LIBYAML_CFLAGS "${LIBYAML_CFLAGS}")
   string (REPLACE ";" " " LIBYAML_LDFLAGS "${LIBYAML_LDFLAGS}")
   if (NOT BUILD_SHARED_LIBS)
-    set (PC_REQUIRES "${PC_REQUIRES} yaml")
+    set (PC_REQUIRES "${PC_REQUIRES} yaml-0.1")
   endif ()
 else ()
   set (LIBYAML_SRC libyaml/yaml.c)
